### DISCUSSION TITLE
No OCR

### DISCUSSION THREAD

**Discussion by Nuube** [papermerge/papermerge-core#315] [General]
I just deployed my first papermerge instance in a docker container as proposed

` docker run -p 8030:8000 -e PAPERMERGE__MAIN__SECRET_KEY=abc  -e DJANGO_SUPERUSER_PASSWORD=123    papermerge/papermerge:latest`

I can log in and add some PDF, but no matter what I do, no OCR is happening. Not automatically, neither when I start the OCR manually. The OCR status indicator always stays grey.

What do I miss?

**Comment by ciur** [comment]
It is because you start only the web app part. For OCR you need to start the worker as well.
There can be only one web app part but as many workers you want.

**Comment by Nuube** [comment]
Sorry, I am quite a noob on these things. I'd had assumed that everything is now in one container. If I start the worker in another container, how would they communicate? Does this mean I have to use the docker-compose setups to include the OCR and only the single container?

**Comment by 360path** [comment]
> It is because you start only the web app part. For OCR you need to start the worker as well. There can be only one web app part but as many workers you want.

How do I start the worker if the web app part is running?

**Comment by ciur** [comment]
In order to have full setup you need web app + worker.
